survey/README.md

```diff
@@ -0,0 +1,12 @@
+While the data is not provided here, to replicate the analysis and make way too many plots, you would:
+
+- Download survey data from https://zenodo.org/records/11150653 and place it in a `survey/raw_data` folder
+- From this directory, run `. ./performAnalysis.sh`
+
+Script descriptions:
+- `combineSurveyData.py` can take data from `../raw_data` and combine it into one file
+- `transformSurveyData.py` converts the categorical data that was just combined intro numerical data
+- `analyzeMaxDiffUtilities.py` takes the transformed data and estimates multinomial logit MaxDiff utilities from the survey responses
+- `analyzeSurveyData.py` looks for correlations and such in the survey responses
+- `generatePlots.py` makes many Likert and box and whisker plots including Figure 2
+- `prioritiesPlot.py` makes several more plots related to software quality priorities including Figure 3
```

Comment:
When you say Figure 2 / Figure 3, you may want to reference in "the paper" (since we have no reference to the paper anywhere in this directory)